docs/typescript.md

```diff
@@ -0,0 +1,83 @@
+---
+id: typescript
+title: Using TypeScript Tests
+sidebar_label: TypeScript
+---
+
+If you built your automated tests using TypeScript in either Playwright, Puppeteer, TestCafe, or Cypress, you need the following workaround in order to use Testrunner Toolkit.
```

Comment:
I wouldn't quite call it a "workaround". I would say `you need to transpile your Typescript source to use it with TestRunner Toolkit`